content/en/blog/2023/additional-collector-sig.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Additional collector Sig for APAC regions
+linkTitle: Additional APAC friendly meetings for collector # Mandatory, make sure that your short title.
+date: 2023-02-23 # Put the current date, we will keep the date updated until your PR is merged
+author: '[Sean Marciniak](https://github.com/MovieStoreGuy) (Atlassian)'
+---
+
+## Additional collector SIGs for APAC
+
+As more investment grows for the collector from around the world, the ability to
+gather and converse becomes a challenge. To help bring more of maintainers and
+contributors together, two additional times have been added:
```

Comment:
Nits:

- Consider using active voice.
- The word "investment" doesn't seem quite right, and could be misleading. Can you think of another word or rephrase it?

Reply:
Would something like:

```suggestion
As more collector community grows around the world, having the ability to meet with each other is a challenge. 
To help bring more of maintainers and contributors together, two additional times have been added:
```

Be more appropriate?  I apologies in advance, I am not exactly sure what you mean by active voice.

Reply:
Yes, that's great! Some further tweaks -- how about:

> As the collector community grows worldwide, ...

and

> To help more APAC <del>of</del> maintainers and contributors connect, we have created two new meetings times:

(Note "of" is deleted.)

Reply:
Sweet, thank you for your help reviewing this @chalin !